js/MVP/views/landing/Landing.tsx

```diff
@@ -49,119 +44,42 @@ const Landing = (): ReactElement => {
-      ) {
-        return false;
+      if (sykmeldinger.data.length === 0) {
+        return <NoSykmeldingerPanel displayBrev={displayBrev} brevData={brev.data} />;
       }
-    }
-    return true;
-  };
-
-  const displayMotebehov = (): boolean => {
-    if (isLabs()) return true;
-
-    if (motebehov.isIdle || motebehov.isError || !motebehov.data.visMotebehov) return false;
-    if (!moteplanlegger.isError && moteplanlegger.data?.status !== AVBRUTT && !erMotePassert(moteplanlegger.data))
-      return false;
-    if (!brev.isIdle && !brev.isError && brev.data[0]) {
-      const brevHead = brev.data[0];
-      if (brevHead.brevType === brevTypes.INNKALT || brevHead.brevType === brevTypes.ENDRING) return false;
-    }
-
-    return true;
-  };
-
-  const BrevPanel = (): ReactElement => {
-    const brevHead = brev.data ? brev.data[0] : undefined;
-
-    if (brevHead?.brevType === brevTypes.REFERAT) {
-      const date = getLongDateFormat(brevHead.tid);
-      return <MotereferatPanel date={date} />;
-    }
-    return <MoteinnkallelsePanel innkallelse={brevHead} />;
-  };
-
-  const PlanleggerPanel = (): ReactElement => {
-    const modus = getSvarsideModus(moteplanlegger.data, BRUKER);
-    const convertedMotedata = konverterTid(moteplanlegger.data);
-    if (modus === BEKREFTET || modus === MOTESTATUS) {
-      return <MoteplanleggerKvitteringPanel mote={convertedMotedata} modus={modus} />;
-    }
-    return <MoteplanleggerPanel modus={modus} />;
-  };
-
-  const DialogmoteFeaturePanel = (): ReactElement | null => {
-    if (displayBrev()) {
-      return BrevPanel();
-    }
-    if (!moteplanlegger.isError && !erMotePassert(moteplanlegger.data)) {
-      return PlanleggerPanel();
-    }
-    return null;
-  };
-
-  const PreviousMotereferatFeaturePanel = ({
-    displayAlleReferater,
-  }: PreviousMotereferatFeaturePanelProps): ReactElement | null => {
-    if (brev.isIdle || brev.isError || (!displayAlleReferater && brev.data.length < 2)) return null;
-
-    const currentBrev = displayBrev() && !displayAlleReferater ? brev.data.slice(1) : brev.data;
-    const previousReferater = currentBrev.filter((hendelse) => hendelse.brevType === brevTypes.REFERAT);
-    const previousReferatDates = previousReferater.map(({ tid }) => tid);
-
-    return <PreviousMotereferatPanel previousReferatDates={previousReferatDates} />;
-  };
-
-  const MainContentPanel = (): ReactElement => {
-    if (hasNoSendteSykmeldinger()) {
       return (
         <React.Fragment>
-          <IkkeSykmeldtLanding />
-          <PreviousMotereferatFeaturePanel displayAlleReferater={true} />
+          <MotebehovPanel displayPanel={displayMotebehov} motebehov={motebehov} />
+          <BrevPanel displayPanel={displayBrev} brevData={brev.data} />
+          <PlanleggerPanel displayPanel={!isMotePassert} moteplanleggerData={moteplanlegger.data} />
```

Comment:
`PlanleggerPanel` skal heller ikke vises når den er avbrutt OG det er har blitt sendt ut et innkallelsebrev. Sjekker vi dette?

Reply:
hm nei displayPanel burde vel her vært !isMotePassert && !displayBrev for å gjøre det likt som før. eller?

Reply:
Yepp, tror det skal gjøre det likt ja.